packages/formspree-react/tsconfig.json

```diff
@@ -1,17 +1,14 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "@formspree/react",
+  "extends": ["../../tsconfig.json"],
   "compilerOptions": {
-    "target": "es5",
-    "module": "ESNext",
-    "lib": ["es6", "dom"],
     "declaration": true,
     "declarationDir": "dist/types",
-    "noImplicitAny": true,
-    "moduleResolution": "node",
+    "declarationMap": true,
     "jsx": "react",
```

Comment:
Eventually, we should change this to `react-jsx` but it'll produce a breaking change (generated typedef will use jsx transform from `react/runtime` instead).

https://stackoverflow.com/questions/67776707/ts-config-jsx-reactjsx-vs-react